include/mgba-util/common.h

```diff
@@ -51,7 +51,9 @@ typedef intptr_t ssize_t;
 #include <sys/time.h>
 typedef intptr_t ssize_t;
 #else
+#ifndef __CELLOS_LV2__
```

Comment:
Why did this get added?

Reply:
Forgot to rebase my offline files. :| Will get everything cleaned up.

Reply:
I have to rebase my offline work files. Will do that.